build.cake

```diff
@@ -125,6 +125,7 @@ Task("Run-Unit-Tests")
                 parameters.Paths.Files.TestCoverageOutputFilePath,
                 new OpenCoverSettings {
                     ReturnTargetCodeOffset = 0,
+                    Register = "user",
```

Comment:
@gep13 yes that's possible currently it's an Azure Web Site hosting jenkins, have to go up a pricing tier to allow 64bit.

One way around it would be an env var to just skip open cover where it's not supported.